Backend/dev/rest-client/dynamic-offer-driver-app/auto-flow.http

```diff
@@ -1,6 +1,11 @@
 @token1 = {{driver-offer-bpp-auto-token1}}
```

Comment:
@roman-bodavskiy are this changes required to be pushed ?

Reply:
this changes affect only local testing with rest client vs code extension, I was not able to test without them, so can be pushed I think